js/store.js

```diff
@@ -192,35 +205,26 @@ const store = {
     }
   },
 
-  getThirdParties(hostname) {
-    if (!hostname) {
-      throw new Error('getThirdParties requires a valid hostname argument');
-    }
-
-    const firstParty = this.getWebsite(hostname);
-    if ('thirdPartyHostnames' in firstParty) {
-      return firstParty.thirdPartyHostnames;
-    }
-
-    return null;
-  },
-
   setWebsite(hostname, data) {
     const websites = clone(this._websites);
```

Comment:
@biancadanforth ugh Github is killing me with hiding comments.

Er well I would fix the other issues first, however if this is an issue you will need to make sure that setFirstParty blocks the thread of the capture code otherwise this will happen. The alternative if we start to worry about this blocking is to write it to another table.

So relationships could be stored in a different table reducing the risk of blocking to the same row. However this is more work that we could leave to be a perf enhancements.